csrc/py_itfs_ck/mha_bwd_kernels.cu

```diff
@@ -315,11 +322,11 @@ mha_bwd(const at::Tensor &dout,         // [b, sq, hq, d]
     at::Tensor dq_accum;
 
     if (!deterministic) {
-        dq_accum = torch::zeros({1, batch_size, seqlen_q, num_heads, head_size}, opts.dtype(at::kFloat));
+        dq_accum = torch::zeros({1, batch_size, num_heads, seqlen_q, head_size}, opts.dtype(at::kFloat));
```

Comment:
this layout is different from q, k, v, dq dk dv. Need to check does it hurt performance in v2